integrations/tensorflow/cpp_projects/dynamic_images/detection_post_process.h

```diff
@@ -0,0 +1,111 @@
+/* Copyright 2022 The MLPerf Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+#ifndef MLPERF_DETECTION_DETECTION_POST_PROCESS_H_
+#define MLPERF_DETECTION_DETECTION_POST_PROCESS_H_
```

Comment:
third party code must go in [third_party/](https://github.com/google/iree/tree/main/third_party/)

The code seems reasonable enough to fork to our license, but it's worth checking the policy (or just author original code without forking)